### PR TITLE
pkcs11_helper: added recipe

### PR DESCRIPTION
**Pull request by alwayslivid**
This is a work-in-progress (WIP) that I came up with at 12:15 AM and includes a bunch of very weird-looking hacks, lacking a fair amount of details and there are some policy violations as well.

Needless to say, I'm aware that this recipe is currently not ready to be merged in its current state.

### CODE REVIEW

File: dev-libs/pkcs11-helper/pkcs11_helper-1.25.1.recipe [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	haiku${secondaryArchSuffix}_devel
+	devel:libssl$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:automake
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize
```
Comment (Begasus) [inline dev-libs/pkcs11-helper/pkcs11_helper-1.25.1.recipe]: Add $secondaryArchSuffix here:
```diff
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
```

**Comment by korli** [comment]
can you provide a build log?

**Comment by Begasus** [comment]
Builds OK here on gcc2h for gcc2/gcc8 ...

**Comment by alwayslivid** [comment]
> Builds OK here on gcc2h for gcc2/gcc8 ...

Please don't merge this quite yet, I still need to add a function in order to save space.

**Comment by Begasus** [comment]
Not merging, just commenting :)

**Comment by alwayslivid** [comment]
Anything else that needs to get done?

**Comment by alwayslivid** [comment]
Successfully built on x86_64.

I'd add a `defineDebugInfoPackage`, but that doesn't seem to be necessary because of a pretty straight forward volume of files. If I'm wrong, I can also add that upon request, it won't take long anyways.

**Comment by korli** [comment]
> can you provide a build log?

**Comment by alwayslivid** [comment]
> > can you provide a build log?

Woops, apologies.

https://paste.ubuntu.com/p/YVk8RRnrGk

**Comment by alwayslivid** [comment]
> > can you provide a build log?

Here's the full build log:

https://paste.ubuntu.com/p/JxBMbnx3Jp/

**Comment by korli** [comment]
Please also remove libpkcs11-helper.a

**Comment by Begasus** [comment]
> Please also remove libpkcs11-helper.a

@korli any specific reason to remove the .a file?

**Comment by korli** [comment]
> @korli any specific reason to remove the .a file?

It uses space and is in the end never used.

**Pull request merged**